kayenta-stackdriver/kayenta-stackdriver.gradle

```
@@ -2,4 +2,6 @@ dependencies {
   implementation project(":kayenta-core")
   implementation project(":kayenta-google")
   api "com.google.apis:google-api-services-monitoring"
+
+  testImplementation 'org.mockito:mockito-inline:2.13.0'
```

The version of this comes from elsewhere.  At the moment, the output of `$ ./gradlew kayenta-stackdriver:dependencies` shows we're getting 3.9.0, with or without a version here.
```suggestion
  testImplementation 'org.mockito:mockito-inline'
```